cpp/src/dictionary/set_keys.cu

```diff
@@ -85,7 +85,6 @@ struct dispatch_compute_indices {
              cudaStream_t stream)
   {
     CUDF_FAIL("list_view dictionary set_keys not supported yet");
-    return nullptr;
```

Comment:
I suspect some compiler versions are going to complain about not having a return value here. Furthermore, this is an unrelated change.

Reply:
IIRC, compiler complains if return type is auto.